public/app/types/alerting.ts

```diff
@@ -40,6 +40,7 @@ export type NotifierType =
   | 'hipchat'
   | 'email'
   | 'sensu'
+  | 'sensugo'
```

Comment:
I am referring to this change.

Reply:
What is referred to simply as 'sensu' in this context is the older version of the Sensu product.  It is also commonly referred to as Sensu Core.  It went EOL in 2019 December but still has a significant number of users.

'sensugo' refers to Sensu Go which is the next generation replacement product that went GA in 2018 December and is in active development.

Reply:
aha, got it! Then it makes sense to name it that way. Sorry for not getting this earlier 🙈

Reply:
I'm wondering if changing [this line](https://github.com/grafana/grafana/blob/3b27c8476aaebe78117ccc0a9aebf2c85b7a7d2e/pkg/services/alerting/notifiers/sensu.go#L17) (and the subsequent one) for the old/original version to "Sensu Core" or "Sensu Core (EOL)" would help in the differentiation.